packages/_private/angular-demo/src/themeswitch/themeswitch.component.ts

```diff
@@ -1,8 +1,6 @@
 import { Component } from '@angular/core';
-import type {
-  SynChangeEvent,
-  SynSwitch as SynSwitchElement,
-} from '@synergy-design-system/components';
+import type { SynChangeEvent } from '@synergy-design-system/angular';
+import type { SynSwitch as SynSwitchElement } from '@synergy-design-system/components';
```

Comment:
Here a problem becomes apparent: We are exporting all types from `@synergy-design-system/components` via `export type *...`.